### PR TITLE
Wekeo support

### PR DESCRIPTION
**Pull request by lforesta**

### CODE REVIEW

File: services/data/data/dependencies/settings.py [hunk 1]
```diff
@@ -140,19 +140,20 @@ def initialise_settings() -> None:
...
-        Validator(SettingKeys.GROUP_PROPERTY.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.WHITELIST.value, must_exist=True, when=not_doc),
-
-        # Validator("CSW_SERVER_DC", must_exist=True, when=Validator("ENV_FOR_DYNACONF", is_not_in=["unittest"]),
-        #           condition=utils.check_url_is_reachable),
-        Validator(SettingKeys.CSW_SERVER_DC.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.DATA_ACCESS_DC.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.GROUP_PROPERTY_DC.value, must_exist=True, when=not_doc),
-        Validator(SettingKeys.WHITELIST_DC.value, must_exist=True, when=not_doc),
+        Validator("IS_CSW_SERVER", default=False),
```
Comment (sophieherrmann) [inline services/data/data/dependencies/settings.py:143]: For documentation reasons each configuration key is defined in the enum class `SettingKeys` 
The name and value of the enum are equal and beneath the key definition should be a doc-string describing the key
Then in the settings validation these keys are used e.g. `SettingKeys.IS_CSW_SERVER.value`

File: services/data/data/dependencies/settings.py [hunk 2]
```diff
@@ -163,9 +164,16 @@ def initialise_settings() -> None:
                   when=not_doc_unittest),
     )
     settings.validators.validate()
+    if not (settings.IS_CSW_SERVER or settings.IS_CSW_SERVER_DC or settings.IS_HDA_WEKEO):
+        raise Exception("No (meta)data connector is specified. At least one of the"
+                        "following env variables must be true: OEO_IS_CSW_SERVER,"
+                        " OEO_IS_CSW_SERVER_DC, OEO_IS_HDA_WEKEO.")
 
-    if settings.ENV_FOR_DYNACONF != "documentation":
+    if settings.IS_CSW_SERVER and settings.ENV_FOR_DYNACONF != "documentation":
```
Comment (sophieherrmann) [inline services/data/data/dependencies/settings.py:172]: It's probably nicer to have one big if for "not documenation" and then the other ones inside (?)

**Review by sophieherrmann — COMMENTED** [review]
Looks good, only the configuration keys were intended a bit differently.

File: services/data/data/dependencies/settings.py (hunk 1 above)
Comment (lforesta) [reply services/data/data/dependencies/settings.py:143]: Perfect, I'll use the SettingKeys then

File: services/data/data/dependencies/settings.py (hunk 2 above)
Comment (lforesta) [reply services/data/data/dependencies/settings.py:172]: hmn yeah, I wanted to avoid nested if cycles, but I can update this

**Pull request merged**